app/code/Magento/QuoteGraphQl/etc/schema.graphqls

```diff
@@ -350,6 +350,7 @@ type CartItemPrices {
 type SelectedCustomizableOption {
     id: Int!
     label: String!
+    type: String!
```

Comment:
this needs a simple test

Reply:
Hi @cpartica , I have some issues while installing PHPUnit in my system. That's why I haven't issued the PR yet. I will soon resolve and give a PR